scripts/run_tests_unit.sh

```diff
@@ -31,6 +31,38 @@ if [ -z "$EQ_ENABLE_CACHE" ]; then
   export EQ_ENABLE_CACHE=True
 fi
 
+if [ -z "$EQ_MINIMIZE_ASSETS" ]; then
+  export EQ_MINIMIZE_ASSETS=False
+fi
+
+if [ -z "$EQ_SECRET_KEY" ]; then
+  export EQ_SECRET_KEY="SuperSecretDeveloperKey"
+fi
+
+if [ -z "$EQ_SERVER_SIDE_STORAGE_USER_ID_SALT" ]; then
+  export EQ_SERVER_SIDE_STORAGE_USER_ID_SALT="developer.user.id.salt"
+fi
+
+if [ -z "$EQ_SERVER_SIDE_STORAGE_USER_IK_SALT" ]; then
```

Comment:
Can this duplication with run_app.sh be removed (e.g. do a `source dev_vars.sh` or something?